docs/sources/alerting/fundamentals/alert-rules/annotation-label.md

```diff
@@ -26,6 +26,12 @@ weight: 105
 
 Labels and annotations contain information about an alert. Labels are used to differentiate an alert from all other alerts, while annotations are used to add additional information to an existing alert.
 
+When creating alert rules, you can leverage labels and annotations to optimize and customize your alerts. **Labels** serve as unique identifiers of an alert. And, beyond their labeling function, they also offer practical functionalities such as searchability, silencing, routing notifications, and they can be templated.
```

Comment:
```suggestion
When creating alert rules, you can leverage labels and annotations to optimize and customize your alerts. 

Labels are unique identifiers of an alert rule. They can also be used for searching, silencing, and routing notifications. 
```

Reply:
You can also template labels and annotations.

Reply:
i don't see this change yet..maybe you missed it in the midst of all the others haha

Reply:
![image](https://github.com/grafana/grafana/assets/45235678/81fd3914-6732-48a6-8497-0ae88d461bfc)
somehow it is not reflected in here but i did push it :)